servers/services/src/main/java/org/projectnessie/services/rest/TreeResource.java

```diff
@@ -418,19 +450,15 @@ protected Hash doOps(
     }
   }
 
-  private static CommitMeta meta(Principal principal, CommitMeta commitMeta)
-      throws NessieConflictException {
-    if (commitMeta.getCommitter() != null) {
-      throw new NessieConflictException(
-          "Cannot set the committer on the client side. It is set by the server.");
-    }
-    String committer = principal == null ? "" : principal.getName();
-    Instant now = Instant.now();
-    return commitMeta.toBuilder()
+  private CommitMeta meta(final CommitMeta referredCommit, Map<String, String> additionalProps) {
```

Comment:
`final` is superfluous

Reply:
The name `referredCommit` is a bit misleading, because we're not _referring_ to it.

Reply:
I'll refactor it to `from`